packages/experimental/Expander/windows/ReactNativeExpander/ExpanderView.cpp

```diff
@@ -0,0 +1,272 @@
+                    expander.Height(propertyValue.AsDouble());
+                    this->Height(propertyValue.AsDouble());
+                } else {
+                  expander.ClearValue(xaml::FrameworkElement::HeightProperty());
+                  this->ClearValue(xaml::FrameworkElement::HeightProperty());
+                }
+            }
+            else if (propertyName == "contentHorizontalAlignment") {
+                if (!propertyValue.IsNull()) {
+                    auto alignment = propertyValue.AsString();
+                    if (alignment == "center") {
+                        expander.HorizontalContentAlignment(xaml::HorizontalAlignment::Center);
+                    }
+                    else if (alignment == "left") {
+                        expander.HorizontalContentAlignment(xaml::HorizontalAlignment::Left);
+                    }
+                    else if (alignment == "right") {
+                        expander.HorizontalContentAlignment(xaml::HorizontalAlignment::Right);
+                    }
+                    else if (alignment == "stretch") {
+                        expander.HorizontalContentAlignment(xaml::HorizontalAlignment::Stretch);
+                    }
+                } else {
+                    expander.ClearValue(xaml::Controls::Control::HorizontalContentAlignmentProperty());
+                }
+            }
+            else if (propertyName == "contentVerticalAlignment") {
+                if (!propertyValue.IsNull()) {
+                    auto alignment = propertyValue.AsString();
+                    if (alignment == "bottom") {
+                        expander.VerticalContentAlignment(xaml::VerticalAlignment::Bottom);
+                    }
+                    else if (alignment == "center") {
+                        expander.VerticalContentAlignment(xaml::VerticalAlignment::Center);
+                    }
+                    else if (alignment == "stretch") {
+                        expander.VerticalContentAlignment(xaml::VerticalAlignment::Stretch);
+                    }
+                    else if (alignment == "top") {
+                        expander.VerticalContentAlignment(xaml::VerticalAlignment::Top);
+                    }
+                } else {
+                  expander.ClearValue(xaml::Controls::Control::VerticalContentAlignmentProperty());
+                }
+            }
+            else if (propertyName == "headerBackground") {
+                this->SetResourceColor(propertyValue, L"ExpanderHeaderBackground");
+            }
+            else if (propertyName == "headerForeground") {
+                this->SetResourceColor(propertyValue, L"ExpanderHeaderForeground");
+            }
+            else if (propertyName == "headerBorderThickness") {
+                auto resDict = expander.Resources();
+                if (!propertyValue.IsNull()) {
+                    auto thickness = xaml::ThicknessHelper::FromUniformLength(propertyValue.AsDouble());
```

Comment:
fyi (not blocking for this PR), we now (0.66) have JSValue readers for Thickness which let you specify a uniform thickness, or individual thicknesses `[1,2,3,4]` or `{left:1, top:2, right:3, bottom:4}`